packages/dam-app-base/src/Editor/SortableComponent.tsx

```diff
@@ -280,12 +280,12 @@ export const SortableComponent = ({
   }, [resources, makeThumbnail, getAdditionalData, setItems]);
 
   const handleDragEnd = useCallback(
-    (event) => {
```

Comment:
fixing some typescript complaints, not sure why these weren't relevant in previous versions.